packages/contract/lib/handlers.js

```diff
@@ -59,10 +59,10 @@ const handlers = {
    * @param  {Object} context   execution state
    * @param  {Object} error     error
    */
-  error: function(context, error) {
+  error: function (context, error) {
     if (!handlers.ignoreTimeoutError(context, error)) {
-      context.promiEvent.eventEmitter.emit("error", error);
-      this.removeListener("error", handlers.error);
+      context.promiEvent.emit("error", error);
+      this.off("error", handlers.error);
```

Comment:
Shouldn't removeListener work here? What was the issue? (@spacesailor24)

Reply:
Our new types for pomievent had `off` and `on` for event handling.